### ISSUE TITLE
Sentry Telegram integrations 

### ISSUE THREAD

**Issue by medikos** [getsentry/sentry#61998]
### Problem Statement

Hi all!
My problem is that I want to insist on alerts being sent to the telegram channel.
This question comes up quite often on the Internet.
There is even a plugin https://github.com/butorov/sentry-telegram
which has not been supported for a long time.
Will such functionality ever be implemented in sentry?

### Solution Brainstorm

If the point is that there is no time for development, then I wonder if it is possible to send a pull request for review

### Product Area

Alerts

**Comment by Dhrumil-Sentry** [comment]
@medikos we don't have any plans to build an integration with Telegram. Have you considered building an internal integration within your Sentry to achieve this?

https://docs.sentry.io/product/integrations/integration-platform/

**Comment by drpancake** [comment]
Please please please add this! A bunch of my dev friends do everything in Telegram now and people are moving away from Slack. We build custom bots for various logs and alerts. Telegram has 1bn+ users.

**Comment by joshlopes** [comment]
I find it strange this not been integrated. There is slack, discord etc. Telegram is probably way way easier to integrate using the bot API which is literally a curl to an endpoint, easy integration + maintaining it and would please a lot of people that don't want to pay or overcomplicate things with just error alerting.

**Comment by vocolboy** [comment]
I sent a pull request and can use my package while waiting :)

https://github.com/tianci-sh/sentry-telegram

**Comment by jslirola** [comment]
Any news? It would be great to merge this PR to improve the integration of our projects.

@Dhrumil-Sentry

**Comment by Dhrumil-Sentry** [comment]
The PR submitted works off a Legacy integration mechanism - project-level plugins which we're trying to move away from in favor of org-level integrations: https://sentry.io/integration-platform/

Any new integration added to Sentry in the last couple of years has been an org-level integration- this is why we can't merge the PR. But this integration is a part of our backlog and we will comment here once we have any timelines

**Comment by GadskyPapa** [comment]
> I find it strange this not been integrated. There is slack, discord etc. Telegram is probably way way easier to integrate using the bot API which is literally a curl to an endpoint, easy integration + maintaining it and would please a lot of people that don't want to pay or overcomplicate things with just error alerting.

I find it really strange too. It's really easy to integrate and telegram is quite popular and functional messenger.

**Comment by mischasigtermans** [comment]
Any news on this?

**Comment by sentaur-athena** [comment]
This integration is built by a member of community and will be published on Sentry soon as a 3rd party integration.

**Comment by ismoil-nosr** [comment]
> This integration is built by a member of community and will be published on Sentry soon as a 3rd party integration.

Hi, do we have any updates on this?

**Comment by sentaur-athena** [comment]
Yes, the community integration is at latest stages of approval and it'll be published soon. I'll update the ticket when I have the exact dates but my guess is some time this week.
